core/src/main/java/jenkins/management/AdministrativeMonitorsConfiguration.java

```diff
@@ -43,7 +43,12 @@
     public boolean configure(StaplerRequest req, JSONObject json) throws FormException {
         for (AdministrativeMonitor am : AdministrativeMonitor.all()) {
             try {
-                boolean disable = !json.getJSONArray("administrativeMonitor").contains(am.id);
+                boolean disable;
+                if(json.optJSONArray("administrativeMonitor") != null) {
```

Comment:
@oleg-nenashev thanks for your comment. optString will not return null, it will return empty string incase there is no key. Please let me know if you still want to make the change as you suggested.